local-server/local-server-services/src/main/resources/fxml/LocalServerServices.fxml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.lang.*?>
+<?import javafx.collections.FXCollections?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.control.TextArea?>
+<?import javafx.scene.layout.BorderPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+
+<BorderPane fx:controller="com.esri.samples.localserver.LocalServerServicesController"
+           xmlns:fx="http://javafx.com/fxml">
+  <!-- Top Section -->
+  <top>
+    <HBox spacing="10">
+      <padding><Insets top="10" right="10" bottom="5" left="10"/></padding>
+    
+      <VBox spacing="3"> 
+        <Label text="Local Server Controls"/>

```

Comment:
I don't think we need this label. The button labels are pretty self-explanatory.